public/locales/en/panel-sessionrecording.json

```diff
@@ -0,0 +1,60 @@
+{
+  "button-labels": {
+    "play": "Play",
+    "pause": "Pause",
+    "resume": "Resume",
+    "stop-playback": "Stop Playback",
+    "record": "Record"
+  },
+  "play-session": {
+    "title": "Play",
+    "loop-playback-label": "Loop playback",
+    "output-frames": {
+      "label": "Output frames",
+      "tooltip": "If checked, the specified number of frames will be recorded as screenshots and saved to disk. Per default, they are saved in the user/screenshots folder. This feature can not be used together with '$t(play-session.loop-playback-label)'."
```

Comment:
I think tooltips are generally used for the hover-info box we have? maybe call this infobox instead? Otherwise there is a mismatch what tooltip means

Reply:
I've been using tooltips for those pop-up info boxes as well. I think it's easier to convey what they are to a translator.